gkeys-gen/gkeygen/actions.py

```diff
@@ -149,7 +175,7 @@ def genkey(self, args):
             ack = py_input("Continue?[y/n]: ").lower()
         if ack in ["n", "no"]:
             messages.extend(['', "\nKey generation aborted."])
-            return (False. messages)
+            return (False, messages)
         elif ack in ["y", "yes"]:
```

Comment:
Indeed this one has been fixed [already](https://gitweb.gentoo.org/proj/gentoo-keys.git/commit/?id=73734eebcef4a9222c170375a3745d19c74f614e). I have contacted with a3li so we can figure what happened.